### DISCUSSION TITLE
How to set line spacing of MarkupText

### DISCUSSION THREAD

**Discussion by CatBraaain** [3b1b/manim#1519] [Q&A] [answered]
Hello, I'm new on Git Hub and I didn't find a solution of the following problem :
I want to set line spacing of MarkupText

my code is that : and it doesnt work

```
text1 = Text('Hello\nWorld',line_spacing=2).shift(LEFT*2)
text2 = MarkupText('Hello\nWorld',line_spacing=2).shift(RIGHT*2)
self.add(text1,text2)
self.wait(1)
```

![WS000000](https://user-images.githubusercontent.com/84499939/118944767-d29e8c80-b98f-11eb-801f-0b565702fde5.JPG)

Thank you for the responses !

**Comment by TonyCrane** [comment, marked as answer]
I think you are using the community version of manim.
So you're supposed to ask this question on discord of ManimCommunity or open an issue in [ManimCommunity/manim](https://github.com/ManimCommunity/manim)

**Reply by CatBraaain** [reply]
Okay Thanks Tony :)
